src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -373,9 +373,20 @@ def create_digest_session(self, alias, url, auth, headers={}, cookies=None,
             debug,
             disable_warnings)
 
-    def create_client_cert_session(self, alias, url, headers={}, cookies=None,
-                       client_certs=None, timeout=None, proxies=None,
-                       verify=False, debug=0, max_retries=3, backoff_factor=0.10, disable_warnings=0):
+    def     create_client_cert_session(
```

Comment:
whats happening with the spaces after `def` ? 
```suggestion
    def create_client_cert_session(....
```